apps/shipments/serializers.py

```diff
@@ -16,10 +16,9 @@
 from enumfields import Enum
 from enumfields.drf.serializers import EnumSupportSerializerMixin
 from jose import jws, JWSError
-from rest_framework import exceptions, status, serializers as rest_serializers
+from rest_framework import exceptions, status, serializers
```

Comment:
What's the reasoning for this change? We should be using rest_framework_json_api serializers in almost all cases.
https://django-rest-framework-json-api.readthedocs.io/en/stable/usage.html#serializers

`TrackingDataToDbSerializer` would be one of the few exceptions to this, since it is used to parse json tracking payloads.

Reply:
With `rest_framework_json_api~3.0` I noticed a strange behavior of `serializers.ModelSerializer` where the included resources were serialized with null fields. I tried my best in vain to fix it and then decided to fallback to `rest_framework.serializers`.

Reply:
I see. Is this the issue you were having?
https://github.com/django-json-api/django-rest-framework-json-api/issues/708

It looks like the current recommendation is to use ResourceRelatedField. We can use that and then define the serializers in the `included_serializers` section.
https://django-rest-framework-json-api.readthedocs.io/en/stable/usage.html#included

I would prefer to stick with the recommended approach so we maintain compatibility with this library moving forward. But if you've already tried this to no avail, then we might have to evaluate other workarounds.

Reply:
Indeed I tried this approach without success.

Reply:
@ClovisDj I had to work through these issues in Profiles for my upgrade to 2.8  I am currently reviewing your PR there and may have additional information coming shortly.